### ISSUE TITLE
Shortcut commands in Tooltips missing

### ISSUE THREAD

**Issue by shrey-sachdeva2000** [accordproject/markdown-editor#258]
**Is your feature request related to a problem? Please describe.**
- With the addition of shortcut commands, they should also be mentioned in `toolbarTooltip` 
- Make the Tooltip more descriptive

**Describe the solution you'd like(reference)**
![image](https://user-images.githubusercontent.com/47667325/75786757-c7137580-5d8b-11ea-8645-61982fb6a286.png)

**Additional context**
- This feature continues #124
- The shortcut commands should be based on OS of user(Ctrl for Windows, ⌘ for Mac)
